packages/apps/testbench-app/src/components/Main.tsx

```diff
@@ -24,19 +24,19 @@ import { exportData, importData } from '../util';
 
 export const Main = () => {
   const client = useClient();
-  // TODO(wittjosiah): Why filter out the default space?
+  // Filter default so that the first space visible is the shared space.
```

Comment:
Note that there's nothing currently stopping the default space from being shared as well